altair/vegalite/v5/theme.py

```diff
@@ -14,43 +15,8 @@
     else:
         from typing_extensions import TypeAlias
 
-    # If you add a theme here, also add it in `VEGA_THEMES` below.
-    _ThemeName: TypeAlias = Literal[
-        "default",
-        "carbonwhite",
-        "carbong10",
-        "carbong90",
-        "carbong100",
-        "dark",
-        "excel",
-        "fivethirtyeight",
-        "ggplot2",
-        "googlecharts",
-        "latimes",
-        "opaque",
-        "powerbi",
-        "quartz",
-        "urbaninstitute",
-        "vox",
-    ]
-
-# If you add a theme here, also add it in `_ThemeName` above.
-VEGA_THEMES = [
-    "carbonwhite",
-    "carbong10",
-    "carbong90",
-    "carbong100",
-    "dark",
-    "excel",
-    "fivethirtyeight",
-    "ggplot2",
-    "googlecharts",
-    "latimes",
-    "powerbi",
-    "quartz",
-    "urbaninstitute",
-    "vox",
-]
+AltairThemes: TypeAlias = Literal["default", "opaque"]
+VEGA_THEMES: list[str] = list(get_args(VegaThemes))
```

Comment:
FYI @binste this combines both of our comments https://github.com/vega/altair/pull/3516#discussion_r1703175478